### PR TITLE
Expand `Beatmap/Beatmap collaborations`

### PR DESCRIPTION
**Pull request by TicClick**
honestly I feel that after this the [collab information guide](https://github.com/ppy/osu-wiki/blob/master/wiki/Guides/Collab_Information/en.md) can be removed -- it has a lot of odd/self-evident bits and overlaps significantly with what I wrote

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Beatmap/Beatmap_collaborations/en.md [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
-**Beatmap collaborations**, or ***collabs***, are beatmap works created by multiple people. The word *Collab* may refer to either a single [difficulty](/wiki/Beatmap/Difficulty), or a [beatmap](/wiki/Beatmap) with [guest difficulties](/wiki/Beatmap/Guest_difficulty).
+**Beatmap collaborations**, or ***collabs***, are single [difficulties](/wiki/Beatmap/Difficulty) or [beatmaps](/wiki/Beatmap) with [guest difficulties](/wiki/Beatmap/Guest_difficulty) created by multiple people. While the term *collab* usually refers to beatmaps, it also means any other media created by several participants, such as [storyboards](/wiki/Storyboard) or songs.
 
-Collab difficulties are usually marked by "Collab" or possession of multiple usernames in the difficulty name.
+## Description
+
+Beatmap collaborations usually have the "Collab" word or possession of multiple usernames in their difficulty name or [beatmap description](/wiki/Beatmap/Beatmap_description), although it is not a strict [ranking criteria](/wiki/Ranking_Criteria) requirement. It is also common for frequent participants to invent amalgamations of their usernames[^name-example] and use them consistently throughout the mapping career.
```
Comment (Walavouchey) [inline wiki/Beatmap/Beatmap_collaborations/en.md:15]:
```suggestion
Beatmap collaborations usually have the word "Collab" or multiple usernames attached to their difficulty name or [beatmap description](/wiki/Beatmap/Beatmap_description), although it is not a strict [ranking criteria](/wiki/Ranking_Criteria) requirement. It is also common for frequent participants to invent amalgamations of their usernames[^name-example] and use them consistently throughout their mapping career.
```

File: wiki/Beatmap/Beatmap_collaborations/en.md [hunk 2]
```diff
@@ -1,11 +1,23 @@
...
+
+The mapping usually starts with the [beatmap host](/wiki/Beatmap/Beatmap_host) splitting the pre-[timed](/wiki/Beatmapping/Timing) beatmap template into several parts[^collab-split] and assigning the said parts among the collaborators according to a shared agreement. Mappers complete[^blind-collab] their parts by laying out the [hit objects](/wiki/Gameplay/Hit_object), applying [hitsounds](/wiki/Beatmapping/Hitsound), using [inherited timing points](/wiki/Client/Beatmap_editor/Timing#inherited-timing-point) where necessary, and then pass their work back to the host, who then combines the changes into a single difficulty by carefully editing the difficulty's [`.osu` file](/wiki/Client/File_formats/Osu_(file_format)).
```
Comment (Walavouchey) [inline wiki/Beatmap/Beatmap_collaborations/en.md:17]:
```suggestion
The mapping usually starts with the [beatmap host](/wiki/Beatmap/Beatmap_host) splitting a pre-[timed](/wiki/Beatmapping/Timing) beatmap template into several parts[^collab-split] and assigning them to the collaborators according to some agreed-upon distribution. Mappers complete[^blind-collab] their parts by laying out the [hit objects](/wiki/Gameplay/Hit_object), applying [hitsounds](/wiki/Beatmapping/Hitsound), using [inherited timing points](/wiki/Client/Beatmap_editor/Timing#inherited-timing-point) where necessary, and then pass their work back to the host, who then combines the changes into a single difficulty by carefully editing its [`.osu` file](/wiki/Client/File_formats/Osu_(file_format)).
```

**Comment by Walavouchey** [comment]
> honestly I feel that after this the [collab information guide](https://github.com/ppy/osu-wiki/blob/master/wiki/Guides/Collab_Information/en.md) can be removed

can agree

File: wiki/Beatmap/Beatmap_collaborations/en.md [hunk 3]
```diff
@@ -1,11 +1,23 @@
...
+
+## Notes and references
+
+[^name-example]: ["masazi — mayday B", a beatmap by Vass_Bass](https://osu.ppy.sh/beatmapsets/74277#osu/233068)
+[^collab-split]: The song is marked up using [timeline](/wiki/Client/Beatmap_editor/Timelines) bookmarks according to its musical structure. Relatively long segments, such as verses, are usually split after every 4 or 8 [measures](/wiki/Music_theory/Measure).
+[^blind-collab]: Although mappers usually rely on the parts already completed to make gameplay transitions as smooth as possible, there exist so-called blind collabs such as [Mitose Noriko - Unreal Player](https://osu.ppy.sh/beatmapsets/828563#osu/1735958) hosted by ::{ flag=DE }:: [Mao](https://osu.ppy.sh/users/2204515).
```
[the remark this answers is not in the document]
Comment (TicClick) [reply wiki/Beatmap/Beatmap_collaborations/en.md:23]: dunno they read as side comments to me. maybe `collab-split` may be properly included into the body but I don't know how

**Review by Walavouchey — DISMISSED** [review]
merge at your discretion

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**